### DISCUSSION TITLE
[PSD10] Exercise 2.40

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#166] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210060678-77a2117b-c4a1-4470-b8f2-57bc62811b24.png)

**Comment by Elia-Belli** [comment]
Seguendo i percorsi che restituiscono 1 è possibile ricavare la funzione del circuito, per minimizzarla si può procedere con l'algebra booleana o una K-map.

In questo caso dal circuito si nota che l'output è 1 quando si seleziona 00 nel primo mux e 01 o 10 nel secondo e quando si seleziona 00 nel secondo: $Y = \bar A \bar B + \bar A B \bar C \bar D + A \bar B \bar C \bar D \rightarrow  \bar A \bar B + \bar B \bar C \bar D + \bar A \bar C \bar D$

**Reply by neoSnakex34** [reply]
corretto

**Comment by fraxb04** [comment]
corretto

**Comment by FeddyLix17** [comment]
@Elia-Belli a me la soluzione viene leggermente diversa, elenco qui di seguito i passaggi che ho effettuato per capire dove magari sto sbagliando
A&#x0304;B&#x0304; +A&#x0304;BC&#x0304;D&#x0304; + AB&#x0304;C&#x0304;D&#x0304;
A&#x0304;(B&#x0304;+BC&#x0304;D&#x0304;) + AB&#x0304;C&#x0304;D&#x0304;
A&#x0304;(B&#x0304; + C&#x0304;D&#x0304;) + AB&#x0304;C&#x0304;D&#x0304;
A&#x0304;B&#x0304; + A&#x0304;C&#x0304;D&#x0304; + AB&#x0304;C&#x0304;D&#x0304;
A&#x0304;B&#x0304; + A&#x0304;C&#x0304;D&#x0304; (1 + B&#x0304;)
A&#x0304;B&#x0304; + A&#x0304;C&#x0304;D&#x0304;
A&#x0304;B&#x0304; + A&#x0304;C&#x0304;D&#x0304;

**Comment by Elia-Belli** [comment]
@FedVlogger17 alla riga 5 hai raccolto $\bar A \bar C \bar D$ ma il secondo termine ha $A$ e non $\bar A$

**Comment by FeddyLix17** [comment]
corretto, grazie!
